phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixConnection.java

```diff
@@ -367,7 +368,7 @@ public ReadOnlyProps getProps() {
             // We do not limit the metaData on a connection less than the global
             // one,
             // as there's not much that will be cached here.
-            Pruner pruner = new Pruner() {
+            PMetaData.Pruner pruner = new PMetaData.Pruner() {
```

Comment:
Is this pruner used anymore?

Reply:
It seems that it is not. I will remove it.